includes/CAPx/Drupal/Importer/Orphans/EntityImporterOrphans.php

```diff
@@ -331,6 +337,45 @@ public function getResults() {
   // ///////////////////////////////////////////////////////////////////////////
 
 
+  /**
+   * @param $entityIds
+   */
+  protected function processMultipleImporterOrphans($entityIds) {
+    $importer = $this->getImporter();
+    $entityType = $importer->getEntityType();
+    $options = $this->getImporterOptions();
+    $action = $options['orphan_action'];
+
+    // Do nothing John Snow.
+    if ($action == "nothing") {
+      return;
+    }
+
+    foreach ($entityIds as $entityId) {
+      $entity = entity_load($entityType, array($entityId));
+      $profile = entity_metadata_wrapper($entityType, $entity[$entityId]);
+
+      switch ($action) {
+        case 'delete':
+          $profile->delete();
+          break;
+
+        case 'block':
+        case 'unpublish':
```

Comment:
Block and unpublish are the same action when it comes to setting the status of the entity. Block is for users and unpublish for nodes. Stacking them together here saves me writing more code.